### ISSUE TITLE
`workspaceFolder` isn't implemented

### ISSUE THREAD

**Issue by fahhem** [loft-sh/devpod#846]
**What happened?**  
If a devcontainer.json specifies a `workspaceFolder` that isn't the home directory nor `/workspaces/$workspaceName`, then `devpod ssh` always lands in `/home/$username`.

Example that I followed: https://code.visualstudio.com/remote/advancedcontainers/change-default-source-mount

**What did you expect to happen instead?**  
`devpod ssh` should land in `workspaceFolder` in the container

**How can we reproduce the bug?** (as minimally and precisely as possible)  

My *`devcontainer.json`*:
```
{
    "name": "...",
    "workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind",
    "workspaceFolder": "/workspace"
}
```

**Local Environment:**  
- DevPod Version: 0.4.2
- Operating System: linux
- ARCH of the OS: AMD64

**DevPod Provider:**  
- Local/remote provider: docker

**Anything else we need to know?**

**Comment by pascalbreuninger** [comment]
Thanks for reporting the issue @fahhem, should be fixed with #855

**Comment by fahhem** [comment]
Great, thank you!!
